src/main/java/redis/clients/jedis/PendingEntry.java

```diff
@@ -0,0 +1,57 @@
+package redis.clients.jedis;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+public class PendingEntry implements Serializable{
```

Comment:
How about `StreamPendingEntry` instead of `PendingEntry`?

Reply:
I wanted to keep it close to EntryID, do you think it's better?